queries/objdump/injections.scm

```diff
@@ -0,0 +1,10 @@
+((comment) @injection.content
+ (#set! injection.language "objdump"))
```

Comment:
It might be more efficient to parse the comments initially than to self-inject.

Reply:
I'm still not sure exactly what is meant by self-inject but if you mean "parse the comments for objdump's own purposes" then it's already doing that currently.

See [grammar.js](https://github.com/ColinKennedy/tree-sitter-objdump/blob/f002f632821d8006444d681abef0f024a9ef5cfe/grammar.js#L100-L106), [a full example parse](https://github.com/ColinKennedy/tree-sitter-objdump/blob/ee8f14c75e47eb24a2a24cea1ed47b4240e64da6/test/corpus/comment.txt#L53-L59) and [a more concise parse](https://github.com/ColinKennedy/tree-sitter-objdump/blob/ee8f14c75e47eb24a2a24cea1ed47b4240e64da6/test/corpus/comment.txt#L25-L26)

In which case I guess this injection can be removed. Just wanted to clarify in case there was a misunderstanding.

Reply:
Yea this can be removed, it's meant for stuff the parser doesn't handle, and in most normal languages a comment can have "comment" syntax - TODOs, FIXMEs, etc which you would inject the comment parser here then - but objdump won't (more like shouldn't?) have user comments

You *could* inject C/C++/Rust/Go in the demangled symbol, but detecting which language it is wouldn't be easy, so it might just be better to leave it as is

Reply:
Thanks for the confirmation. Removed!